doc/rados/configuration/ceph-conf.rst

```diff
@@ -611,30 +617,35 @@ will report the value of a single option.
 
 
 
-Changes since nautilus
+Changes since Nautilus
 ======================
 
-We changed the way the configuration file is parsed in Octopus. The changes are
-listed as follows:
+With the Octopus release We changed the way the configuration file is parsed.
+These changes are as follows:
 
-- repeated configuration options are allowed, and no warnings will be printed.
-  The value of the last one wins. Before this change, we would print out warning
-  messages at seeing lines with duplicated values, like::
+- Repeated configuration options are allowed, and no warnings will be printed.
+  The value of the last one is used, ie. the last one wins. Before this change,
```

Comment:
s/ie./i.e./

But, really, in a context like this, I'd say "The value of the last one is used, which means that the last one in the list is the one that takes effect."

I totally get what's meant by "the last one wins" and I even think that it's saucier and more fun for native speakers. But I'm also imagining a non-native speaker with weak English when I make this less-fun suggestion.

I am being super nit-picky.  But anyway, "ie." isn't a think and "i.e." is.

Reply:
Ack.  Sometimes I'm too generous wrt carrying forward existing wording.